src/components/Weather.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import styled from "@emotion/styled/macro";
+import { MIN_WIDTH_BREAKPOINTS } from "../enums";
+
+const { REACT_APP_WEATHER_API_KEY } = process.env;
+
+const [, , , , , , BETWEEN_SMALL_DEVICES_TABLET_UP] = MIN_WIDTH_BREAKPOINTS;
+
+const WeatherContainer = styled.div`
+  display: flex;
+  align-items: center;
+  text-transform: capitalize;
+  justify-content: center;
+
+  @media only screen and (min-width: ${BETWEEN_SMALL_DEVICES_TABLET_UP}px) {
+    justify-content: flex-start;
+  }
+`;
+
+const WeatherIcon = styled.img`
+  margin-right: 0.3em;
+  height: 1em;
+`;
+
+const Temperature = styled.span`
+  margin-right: 0.5em;
+`;
+
+class Weather extends Component {
+  state = {
+    temperature: null,
+    condition: {
+      sunny: false,
+      cloudy: false,
+      overcast: false,
+      rainy: false,
+      snowy: false
+    }
+  };
+
+  getForecast = async () => {
+    const response = await fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?zip=11354,us&appid=${REACT_APP_WEATHER_API_KEY}`
```

Comment:
Could be moved into a constant outside of the class so that the template literal isn't evaluated per call.

For instance:

```javascript
// ...

const WEATHER_API_URL = `https://api.openweathermap.org/data/2.5/forecast?zip=11354,us&appid=${REACT_APP_WEATHER_API_KEY}`;

// ...

class Weather extends Component {
  // ...

  getForecast = async () => {
    const response = await fetch(WEATHER_API_URL);
    // ...
  }

  // ...
}

// ...
```